cibuildwheel/docker_container.py

```diff
@@ -57,39 +80,49 @@ def __enter__(self) -> "DockerContainer":
             network_args = ["--network=host"]
 
         shell_args = ["linux32", "/bin/bash"] if self.simulate_32_bit else ["/bin/bash"]
+
         subprocess.run(
             [
-                "docker",
+                self.container_engine,
                 "create",
                 "--env=CIBUILDWHEEL",
                 f"--name={self.name}",
                 "--interactive",
-                "--volume=/:/host",  # ignored on CircleCI
                 *network_args,
-                *cwd_args,
+                # Z-flags is for SELinux
+                "--volume=/:/host:Z",  # ignored on CircleCI
```

Comment:
Hmm, this is a good catch. I'll need to run more tests and look further into this. Given these docs, blindly using :Z seems like a bad idea. I'm not sure if this will need an additional option.

Reply:
One option could be to disable the host mount on SELinux? It's already conditional on where the docker is running.

Reply:
I've been testing modifications I'm making here on the project which is my use case for working on this PR: https://gitlab.kitware.com/computer-vision/kwimage_ext/-/merge_requests/4

It seems that not having the `:Z` is no longer breaking me (or maybe it never was to begin with...), either way I just reverted it to the state without this patch.